### PR TITLE
Improve rem/em consistency

### PR DESCRIPTION
**Pull request by zackhall**

### CODE REVIEW

File: assets/css/_default.scss [hunk 1]
```diff
@@ -119,11 +119,11 @@ header {
...
 
     h1 {
-      font-size: 1.4em;
+      font-size: 1.4rem;
     }
 
     h2 {
-      font-size: 1em;
+      font-size: 1rem;
     }
```
Comment (zackhall) [inline assets/css/_default.scss]: Consistent heading sizes, no matter the parent container.

File: assets/css/_default.scss [hunk 2]
```diff
@@ -229,7 +229,7 @@ ul.chevron li {
 
 .container.main {
   background: $foreground;
-  box-shadow: 0 4px 1.5em -0.25em $shadow;
+  box-shadow: 0 4px 1.5rem -0.25rem $shadow;
   margin-bottom: 2rem;
```
Comment (zackhall) [inline assets/css/_default.scss]: Consistent shadow sizes

File: assets/css/_home.scss [hunk 3]
```diff
@@ -10,7 +10,7 @@ html.home {
   }
 
   pre.intro {
-    font-size: 1.2em;
+    font-size: 1.2rem;
   }
```
Comment (zackhall) [inline assets/css/_home.scss]: These could go either way I thought. In the context of this website, it doesn't change anything from the visual standpoint. From a code standpoint, though, I swapped for easier mental math.

File: assets/css/_docs.scss [hunk 4]
```diff
@@ -182,7 +182,7 @@ html.docs {
     input {
       border: none;
       border-bottom: 2px solid $defaultLiteColor;
-      font-size: 0.8em;
+      font-size: 0.8rem;
       padding: 10px 0;
```
Comment (zackhall) [inline assets/css/_docs.scss]: Consistent input font sizes

**Comment by jdalton** [comment]
:metal:

**Comment by jdalton** [comment]
Thank you!

**Pull request merged**